### ISSUE TITLE
Density-Based Spatial Clustering of Applications with Noise

### ISSUE THREAD

**Issue by aishanii** [girlscript/winter-of-contributing#6952]
### Description

If you had referred any resources, add them up in "DS Resources". Similarly, if you had used datasets, include them in "DS Datasets".

This issue will help in getting a better understanding of **'Density-Based Spatial Clustering of Applications with Noise'** in a detailed manner.

It's created on contributors request as mentioned in Discord channel.

**Domain: Machine Learning**

Mentors Assigned : Aishani Singh | Dhruv Bajaj

**Points to Note :**

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title" should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your journey ahead. 🌷

### Domain

Datascience with Python

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by HastiSutaria** [comment]
Name = Hasti Sutaria 
Type = documentation
Approach = Practical implementation and theortical overview

**Comment by HastiSutaria** [comment]
I Would like to work on this in documentation format.

**Comment by prathimacode-hub** [comment]
Have you worked on changes addressed to your PR? @HastiSutaria

**Comment by HastiSutaria** [comment]
Yes @prathimacode-hub. I made the required changes yesterday itself

**Comment by HastiSutaria** [comment]
Also ma’am once you approve it. I am ready with another pr completed from my side is data science domain. So I will also create its pr.

**Comment by prathimacode-hub** [comment]
Sure, you can proceed. @HastiSutaria

**Comment by kishlayaug15** [comment]
Name - Kishlay Kumar
Batch - 4
Serial No - 673
GithubLink :- https://github.com/kishlayaug15
Approach: A detailed explanation with Audio

**Comment by prathimacode-hub** [comment]
You had assigned to other issue also i think? @kishlayaug15

**Comment by HastiSutaria** [comment]
@prathimacode-hub detailed explaination in READme file and practical implementation in ipynb file would be enough ma'am for this issue ?

**Comment by HastiSutaria** [comment]
@prathimacode-hub Ma'am can I please get the exact path under Machine learning branch where to upload file while creating the PR?

**Comment by prathimacode-hub** [comment]
Yes, it would be enough. It will go under "Datascience_with_Python/Machine Learning/Tutorials" folder  @HastiSutaria
